tests/Rector/Property/TypedPropertyFromColumnTypeRector/Fixture/simple_column.php.inc

```diff
@@ -25,7 +25,7 @@ class SimpleColumn
     /**
      * @ORM\Column(type="string")
      */
-    private ?string $name = null;
+    private string $name;
```

Comment:
per noted at https://github.com/rectorphp/rector/issues/7709#issuecomment-1378912013 , the rule should be configurable instead, as except `#[ORM\Id], non-initiable property will got unitialized error quickly

Reply:
IMHO nullable:false Is default, so it should be also respected by property typehint. Otherwise it Is inconsistency between ORM definition And typehint.

Reply:
Entity ID is magic, that's why allowed to be typed even never assigned inside class, other properties are not, the entity usage can be abused and cause random initialized error. 

As far as the rule is configurable to allow enable fully typed based on docblock (disable by default), it is ok.

But make a typed property based on doc is incorrect generally, we seen a lot of legacy app that rely on docblock and never initialized, changing to typed broke everything.